app/templates/components/forms/orders/order-form.hbs

```diff
@@ -81,52 +81,103 @@
+              <i class="dropdown icon"></i>
+              <div class="menu">
+                {{#each countries as |country|}}
+                  <div data-value="{{map-value mapper country.name}}" class="item">
+                    <i class="{{lowercase country.code}} flag"></i>
+                    {{country.name}}
+                  </div>
+                {{/each}}
+              </div>
+            {{/ui-dropdown}}
+          </div>
+        {{else}}
+          <div class="ui center aligned field">
+            {{ui-checkbox label=(t 'Enable Billing Information') checked=data.isBillingEnabled
+                                              onChange=(action (mut data.isBillingEnabled))}}
+          </div>
+          {{#if data.isBillingEnabled}}   
+            <div class="field">    
+              <label for="company">{{t 'Company'}}</label>    
+              {{input type='text' id='company' value=data.company}}    
+            </div>   
+            <div class="field">    
+              <label class="required" for="taxBusinessInfo">{{t 'Tax ID or Business ID'}}</label>   
+              {{input type='text' id='taxBusinessInfo' value=data.taxBusinessInfo}}    
+            </div>   
+            <div class="field">    
+              <label class="required" for="address">{{t 'Address'}}</label>    
+              {{textarea rows='2' id='address' value=data.address}}    
+            </div>   
+            <div class="field">    
+              <label class="required" for="city">{{t 'City'}}</label>    
+              {{input type='text' id='city' value=data.city}}    
+            </div>   
+            <div class="field">    
+              <label for="state">{{t 'State/Province'}}</label>    
+              {{input type='text' id='state' value=data.state}}    
+            </div>   
+            <div class="field">    
+              <label class="required" for="zipcode">{{t 'Zip Code'}}</label>   
+              {{input type='text' id='zip_code' value=data.zipcode}}   
+            </div>   
+            <div class="field">    
+              <label class="required">{{t 'Select country'}}</label>   
+              {{#ui-dropdown class="search selection" selected=data.country    
+                            onChange=(action (mut data.country)) as |execute mapper|}}
+                {{input type='hidden' id='country' value=data.country}} 
+                <div class="default text">{{t 'Select country'}}</div>   
+                <i class="dropdown icon"></i>    
+                <div class="menu">   
+                  {{#each countries as |country|}}   
+                    <div data-value="{{map-value mapper country.name}}" class="item">    
+                      <i class="{{lowercase country.code}} flag"></i>    
+                      {{country.name}}   
+                    </div>   
+                  {{/each}}    
+                </div>   
+              {{/ui-dropdown}}   
+            </div>   
+          {{/if}}
```

Comment:
That's duplication. Otherwise known as 'Mother of Bugs' 

Remove any duplication

Reply:
Will remove the duplicated code by tonight

Reply:
There's still repetition

Reply:
Since we need to render two seperate checkboxes, Altogether I have removed the duplication of forms. Also, We need to toggle the value of `is_billing_enabled` in other `else` part, hence I dont think it would be duplication. 😕

Reply:
Will give you a solution

Reply:
```
if not billing_info_mandatory:
    show_checkbox
if billing_info_mandatory or is_billing_enabled:
    show billing form
```